### PR TITLE
Add install/uninstall overview for Ubuntu PWA

### PR DESCRIPTION
**Pull request by pzrq**
### Description

Add install/uninstall overview confirmed to work for Ubuntu PWA.

### Motivation

Happy for others to see if it works in earlier browser versions than v111, or earlier operating systems, or is otherwise confirmed to be supported upstream by those systems/browser vendors. I just put down my test setup right now (Ubuntu 22.04.2 LTS, Brave 111/Edge 111) since I can verify it worked for me.

### Additional details

I started looking as this is the route Microsoft Teams on Linux has gone down, so that's at least one major vendor now supporting the PWA experience: https://techcommunity.microsoft.com/t5/microsoft-teams-blog/microsoft-teams-progressive-web-app-now-available-on-linux/ba-p/3669846

### CODE REVIEW

File: files/en-us/web/progressive_web_apps/installing/index.md [hunk 1]
```diff
@@ -77,3 +79,11 @@ Regardless of which browser and device you're using, when you choose to add the
 ![Screenshot of an Android home screen with the "Foxes" app displayed](a2hs-on-home-screen.png)
 
 When you tap the web app's icon on the home screen, it opens up in a full screen web environment, without the browser's UI around it.
+
+### Uninstalling
+
```
Comment (estelle) [inline files/en-us/web/progressive_web_apps/installing/index.md:84]: Drive by comment: Thank you for adding "uninstalling". Would be great to include uninstalling instructions for all the browser/device combos on the page.

Comment (pzrq) [reply files/en-us/web/progressive_web_apps/installing/index.md:84]: @estelle Thank you. I definitely do not have access to a recent Android device so am not personally able to complete that.

Another way around could be to take uninstalling out of the document title, as it was already present, so it's at least less misleading?

**Comment by captainbrosset** [comment]
It might be worth keeping in mind here that we're currently rewriting the PWA docs on MDN (see this original project proposal: https://github.com/mdn/mdn/issues/280). And as a result, there are chances that the page being changed in this PR might get deleted/recreated.

@wbamberg we should make a note to move the content into the new guides we're writing.

File: files/en-us/web/progressive_web_apps/installing/index.md (hunk 1 above)
Comment (estelle) [reply files/en-us/web/progressive_web_apps/installing/index.md:84]: I'll add it. Working on it now. I should have a PR ready later today. :D

**Review by estelle — APPROVED** [review]

**Pull request merged**